app/src/androidTest/java/com/sdp13epfl2021/projmag/ProjectInformationActivityTest.kt

```diff
@@ -18,6 +21,7 @@ import androidx.test.filters.LargeTest
 import com.google.firebase.auth.FirebaseAuth
 import com.google.firebase.auth.FirebaseUser
 import com.sdp13epfl2021.projmag.activities.ProjectInformationActivity
+import com.sdp13epfl2021.projmag.activities.WaitingListActivity
```

Comment:
An import was missing
```suggestion
import com.sdp13epfl2021.projmag.activities.ProjectCreationActivity
import com.sdp13epfl2021.projmag.activities.WaitingListActivity
```